### PR TITLE
F UI post meeting fixes

### PR DESCRIPTION
**Pull request by Noam-Moskowitz**

### CODE REVIEW

File: frontend/src/components/Login/Login.tsx [hunk 1]
```diff
@@ -294,7 +288,12 @@ export default function Login({ onLogin }: ILoginProps) {
                 </View>
                 <TouchableOpacity
                   onPress={
-                    isForgotPassword ? () => setIsForgotPassword(false) : chooseDifferentMail
+                    isForgotPassword
+                      ? () => {
+                          setIsForgotPassword(false);
+                          setShowConfirmButton(true);
+                        }
```
Comment (mike16345) [inline frontend/src/components/Login/Login.tsx:295]: put this in a function  if its more than one line.  or make a variable that defines which function to use

File: frontend/src/components/WorkoutPlan/RecordExercise.tsx [hunk 2]
```diff
@@ -124,7 +124,25 @@ const RecordExercise: FC<RecordExerciseProps> = ({ route, navigation }) => {
...
+            ]}
+          >
+            <NativeIcon
+              library="AntDesign"
+              name="exclamationcircleo"
+              style={[fonts.xxxxl, colors.textOnSecondaryContainer]}
+            />
+            <Text style={[colors.textOnSecondaryContainer]}>סרטון לא נמצא</Text>
+          </View>
+        )}
```
Comment (mike16345) [inline frontend/src/components/WorkoutPlan/RecordExercise.tsx:145]: dont use ? : (short-if) statements to return JSX its always messy and hard to read. 

Instead use && and check both conditions

File: frontend/src/components/WorkoutPlan/WorkoutPlan.tsx [hunk 3]
```diff
@@ -82,17 +85,24 @@ const WorkoutPlan: FC<WorkoutPlanProps> = () => {
     await setItem(JSON.stringify(session));
   };
 
-  useEffect(() => {
+  const getWorkoutPlan = () => {
```
Comment (mike16345) [inline frontend/src/components/WorkoutPlan/WorkoutPlan.tsx:88]: We should put this in a react query. Make sure to keep the functionality of when the data is retrieved though

File: frontend/src/components/Login/ConfirmPassword.tsx [hunk 4]
```diff
@@ -29,7 +32,7 @@ const ConfirmPassword: React.FC<ConfirmPasswordProps> = ({
           style={[text.textRight, { width: "100%" }]}
           mode="outlined"
           activeOutlineColor={colors.borderSecondary.borderColor}
-          error={Boolean(errors.password)}
+          error={!!(errors.password || errors.validPassword)}
```
[the remark this answers is not in the document]
Comment (Noam-Moskowitz) [reply frontend/src/components/Login/ConfirmPassword.tsx:35]: it indeed does not, making the change you requested specifically throws a typescript error saying it is not recieving a boolean value

[the remark this answers is not in the document]
Comment (mike16345) [reply frontend/src/components/Login/ConfirmPassword.tsx:35]: ya thats why i said should weird that it doesnt

[the remark this answers is not in the document]
Comment (Noam-Moskowitz) [reply frontend/src/components/Login/ConfirmPassword.tsx:35]: i believe this means we shall leave it as is.

very scrumptious, i bid you farewell my friend.

**Review by mike16345 — APPROVED** [review]

**Pull request merged**